docs/_layouts/docs.html

```diff
@@ -2,11 +2,11 @@
 layout: default
 ---
 
-<div id="toggleMenuWrapper" class="docs-page innerwrapper D(tb)--sm Tbl(f) Pt(20px) Mb(50px) Maw(1000px)--sm Miw(1000px)--lg Mx(a)--sm W(96%)--sm">
+<div id="toggleMenuWrapper" class="docs-page innerwrapper D(tb)--sm Tbl(f) Pt(20px) Mb(50px) Maw(1280px)--sm Miw(1280px)--lg Mx(a)--sm W(96%)--sm">
     <button id="toggleMenuButton" class="menu-button Bgi(hamburger) W(32px) H(32px) D(n)--sm Pos(a) Bdw(0) Bgc(t) P(0) T(0) Start(0) Z(7) M(10px) menu-on_Bgp(end_t)">
         <b class="Hidden">Toggle the menu</b>
     </button>
-    <div id="aside" class="D(tbc) Bgc(#fff) Mt(-20px)--xs Pos(a)--xs Va(t) W(150px)--sm End(0) Pt(20px) Pb(40px) menu-on_Pstart(10px) Pend(50px)--sm Z(5) End(a)--sm Start(0) Pos(a) Pos(r)--sm" role="aside">
+    <div id="aside" class="D(tbc) Bgc(#fff) Mt(-20px)--xs Pos(a)--xs Va(t) W(250px)--sm End(0) Pt(20px) Pb(40px) menu-on_Pstart(10px) Pend(50px)--sm Z(5) End(a)--sm Start(0) Pos(a) Pos(r)--sm" role="aside">
```

Comment:
any benefit in adding this at medium or large?

```
W(150px)--sm W(250px)--md```
```

Reply:
ya it looks better, fixed